docs/getting_started/overview_oh.md

```diff
@@ -45,21 +45,7 @@ The {{site.data.keyword.ieam}} management hub is designed specifically for edge
 # Components
 {: #components}
 
-{{site.data.keyword.edge_notm}} ({{site.data.keyword.ieam}}) includes [several components](../getting_started/components.md) that are bundled with the product. The following table provides a description of the {{site.data.keyword.ieam}} components:
-
-|Component|Version|Description|
-|---------|-------|----|
-|**{{site.data.keyword.ieam}} management hub**|4.2.0|The {{site.data.keyword.ieam}} management hub manages the edge nodes and the edge service software lifecycle on each edge node.|
-|Agbot|2.27.0|Agreement bot (agbot) instances are created centrally and are responsible for deploying workloads and machine learning models to {{site.data.keyword.ieam}} edge nodes.|
-|Exchange|2.54.0|The Exchange provides a REST API to all of the {{site.data.keyword.ieam}} resources (patterns, policies, services, nodes, and so on) used by all the other components in {{site.data.keyword.ieam}}.|
-|MMS|1.4.1|The Model Management System (MMS) facilitates the storage, delivery, and security of models and files needed by edge services. This enables edge nodes to easily send and receive models and files to and from the management hub.|
-|SDO|1.8.6|The Secure Device Onboarding (SDO) service enables SDO-enabled edge devices to be configured with zero touch.|
-|Secrets Manager|1.0.0|The Secrets Manager is the repository for secrets deployed to edge devices, enabling services to securely receive credentials used to authenticate to their upstream dependencies.|
-|**Edge node**| |Any edge device, edge cluster, or edge gateway where edge computing takes place.|
-|Edge cluster agent|2.27.0|The agent that is installed on edge clusters to enable node workload management by {{site.data.keyword.ieam}}.|
-|Edge device agent|2.27.0|The agent that is installed on edge devices to enable node workload management by {{site.data.keyword.ieam}}.|
-|ESS|1.4.1|The edge node part of MMS that makes AI models and files available to the edge services.|
-|Example edge services|2.27.0|Edge service examples that are useful when exploring {{site.data.keyword.ieam}} and learning how to write your own services.|
+{{site.data.keyword.edge_notm}} ({{site.data.keyword.ieam}}) includes [several components](../getting_started/components.md) that are bundled with the product. Please see (Getting Started/Components.md).
```

Comment:
Let's not call Open Horizon a "product".  It's an open source project.
Here is a proposed rewrite of this sentence:
```
For more information about components that are bundled with {{site.data.keyword.ieam}}, see [Components](components.md).
```
Note that both the overview_oh.md and the components.md are in the same ../getting_started/ directory so we don't actually need to use a relative path.